packages/terra-clinical-onset-picker/src/OnsetUtils.js

```diff
@@ -9,16 +9,16 @@ const PrecisionOptions = {
   AFTER: 'after',
   UNKNOWN: 'unknown',
 };
+const DATE_FORMAT = 'YYYY-MM-DD';
 
 class OnsetUtils {
   /*
    * Create object to pass to SelectField for month options
    * Filters out months before birth or after the current date.
    */
-  static allowedMonths(intl, birthdate, onsetDate) {
+  static allowedMonths(intl, birthdate, year) {
     const start = moment(birthdate);
-    const end = moment();
-    const onsetYear = onsetDate ? onsetDate.year() : moment().year();
+    const end = moment(moment.now());
```

Comment:
What's the reason for passing in `moment.now()`? Is this different from just doing `moment()`?

Reply:
Changed c4b4f2f